routes/endpoint-docs/userRoutesDocs.js

```diff
@@ -68,6 +68,11 @@
  * @apiGroup Users
  *
  * @apiParam {Number} _id Users unique ID.
+ * @apiParamsExample  {} Request-Example:
+ *    {
+ *   
+ *    }
```

Comment:
Ok to leave this blank for now while User endpoints are being rebuilt, just remember its there!

Reply:
Sorry I left it the time you said Pavol was working on it